language-server/modules/langserver-core/src/test/resources/codeaction/add-isolated-qualifier/source/isolatedFunctionCodeAction/modules/module2/add_isolated_qualifier_source2.bal

```diff
@@ -2,3 +2,45 @@ isolated function bar() {
     testFunction();
     return;
 }
+
```

Comment:
Is there any perticular reason to only have tests in module?

Reply:
We have the tests in a package so that we can test modifications happening on different files.